exercises/sum.cpp

```diff
@@ -6,3 +6,30 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream> 
+using namespace std; 
+
+int somma (int x, int y){
+	return x+y; 
+}
+
+int Inserire(){
+	int x; 
+	
+	cin>>x; 
+	
+	return x;  
+}
+
+
+int main(){
+	int a, b; 
+	
+	cout<<"inserisci due numeri: "; 
+	a=Inserire(); 
+	b=Inserire(); 
```

Comment:
potresti scrivere
```c++
auto a = Inserire();
auto b = Inserire();
```
in modo da dichiarare a e b quando servono ed inoltre il tipo `auto` equivale a esattamente il tipo di ritorno della funzione `Inserire()` quindi `int`, ma è meglio scrivere `auto` al posto di `int` al fine di definirlo una sola volta in una sola parte di codice

Reply:
Ho provato a farlo ma mi dà errore. Purtroppo non ho capito bene come funziona auto (ho cercato in giro, ma non ho ancora ben chiaro come utilizzarlo).  
La mia situazione è questa: 
![immagine_2022-10-23_160017195](https://user-images.githubusercontent.com/115897748/197396486-a2753bdf-240c-4c87-b778-29c718ceceaa.png)

Reply:
Ho provato a farlo ma mi dà errore. Purtroppo non ho capito bene come funziona auto (ho cercato in giro, ma non ho ancora ben chiaro come utilizzarlo).
La mia situazione è questa: 
![Screenshot (156)](https://user-images.githubusercontent.com/115897748/197396677-dce3b070-5557-4571-8678-98cd6f78adbd.png)